src/data/index.ts

```diff
@@ -66,17 +66,14 @@ export const exportCanvas = async (
     }
   }
 
-  const tempCanvas = await exportToCanvas(elements, appState, files, {
+  const tempCanvas = exportToCanvas(elements, appState, files, {
     exportBackground,
     viewBackgroundColor,
     exportPadding,
   });
-  tempCanvas.style.display = "none";
-  document.body.appendChild(tempCanvas);
```

Comment:
as a side-effect I'm removing appending to DOM which was supposedly fixing RTL issues (https://github.com/excalidraw/excalidraw/pull/1687), but I can't reproduce on export (possibly because we keep appending to DOM during rendering step).

@YounessFkhach can you double check if anything is broken? 🙏